commands/domains.go

```diff
@@ -371,6 +371,7 @@ func RunRecordUpdate(c *CmdConfig) error {
 	if err != nil {
 		return err
 	}
+	fmt.Println(rTTL)
```

Comment:
was this line for testing purposes or is this the line that prints proper output?

Reply:
for testing- it's been removed in the next commit- thanks!